Scripts/Python/xHighLevelStarTrekDoor.py

```diff
@@ -144,219 +118,76 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
         elif VARname == strDoorClosedVar.value:
             doorClosed = ageSDL[strDoorClosedVar.value][0]
             PtDebugPrint("HighLevelStarTrekDoor.OnSDLNotify: Door Closed SDL Updated to: %d" % doorClosed)
-            PtDebugPrint("HighLevelStarTrekDoor.OnSDLNotify: Player who updated SDL: ", playerID)
-            if playerID == 0 and self.sceneobject.isLocallyOwned():
-                if doorClosed == 0:
-                    self.SendNote("respOpenDoor;1")
-                    self.UpdateDoorState(doorSDLstates['open'])
-                elif doorClosed == 1:
-                    self.SendNote("respCloseDoor;1")
-                    self.UpdateDoorState(doorSDLstates['closed'])
+            if doorClosed:
+                respCloseDoor.run(self.key,netPropagate=0)
+            else:
+                respOpenDoor.run(self.key,netPropagate=0)
```

Comment:
Since we're now explicitly doing everything locally, we should probably mark all of the `ptAttribResponder`s as `netPropagate=False` explicitly at the top of the file, then we can remove all of these manual `netPropagate=0`s.